src/Wallabag/CoreBundle/Resources/config/services.yml

```diff
@@ -39,7 +39,22 @@ services:
     wallabag_core.graby:
         class: Graby\Graby
         arguments:
-            - { error_message: "wallabag can't retrieve contents for this article. Please report this issue to us." }
+            -
+                error_message: "wallabag can't retrieve contents for this article. Please report this issue to us."
+                http_client:
+                    user_agents:
+                        'lifehacker.com': 'PHP/5.2'
```

Comment:
Next weeks?
We don't update that list quite often.
But I agree that this isn't greatly designed in graby (inherited from full-text-rss on that part)